Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN pip3 install \
 COPY --from=nodebuild /usr/local/src/app/packages ./packages
 COPY --from=nodebuild /usr/local/src/app/node_modules ./node_modules
 COPY --from=nodebuild /usr/local/src/app/package.json ./package.json
-COPY buildinfo ./buildinfo
+# COPY buildinfo ./buildinfo
```

Comment:
Try this
[`https://stackoverflow.com/questions/31528384/conditional-copy-add-in-dockerfile`](https://stackoverflow.com/questions/31528384/conditional-copy-add-in-dockerfile)